### PR TITLE
Upgrade to Elm-0.19

### PR DESCRIPTION
**Pull request by rofrol**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,8 +12,8 @@ language: node_js
 
 env:
   matrix:
-    - ELM_VERSION=0.18.0 TARGET_NODE_VERSION=node
-    - ELM_VERSION=0.18.0 TARGET_NODE_VERSION=4.8
+    - ELM_VERSION=0.19.0 TARGET_NODE_VERSION=node
+    - ELM_VERSION=0.19.0 TARGET_NODE_VERSION=4.0
```
Comment (Janiczek) [inline .travis.yml]: Why the change to 4.0? We need Yarn to work with that Node, and Yarn supports 4.8+

File: .travis.yml [hunk 2]
```diff
@@ -37,8 +37,8 @@ install:
   - printf '%s\n\n' '#!/bin/bash' 'echo "Running elm-make with sysconfcpus -n 2"' '$TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 elm-old "$@"' > $(npm config get prefix)/bin/elm
   - chmod +x $(npm config get prefix)/bin/elm
   - npm install
-  - pushd tests && elm make --yes && popd
+  - pushd tests && elm make && popd
```
Comment (Janiczek) [inline .travis.yml]: Again, why this revert?

File: package.json [hunk 3]
```diff
@@ -13,8 +13,11 @@
...
     "chokidar-cli": "^1.2.1",
-    "elm-test": "^0.18.13-beta3",
-    "elm-verify-examples": "^2.3.1",
-    "elm": "0.18.0"
+    "elm": "^0.19.0-bugfix2",
+    "elm-test": "^0.19.0-beta11",
+    "elm-verify-examples": "^3.0.0"
+  },
+  "dependencies": {
+    "elm-format": "^0.8.1"
```
Comment (Janiczek) [inline package.json]: This could be devDependency too

File: .travis.yml (hunk 2 above)
Comment (Janiczek) [reply .travis.yml]: Ah, probably because 0.19 `elm make` doesn't have --yes anymore. Fair enough! :+1:

**Comment by turboMaCk** [comment]
@rofrol are you reading slack at all? We already know how to solve this.

File: .travis.yml [hunk 4]
```diff
@@ -28,17 +25,14 @@ before_install:
...
-  - nvm use $TARGET_NODE_VERSION
-  - node --version
   - npm --version
   - npm install -g elm@$ELM_VERSION
   - mv $(npm config get prefix)/bin/elm $(npm config get prefix)/bin/elm-old
   - printf '%s\n\n' '#!/bin/bash' 'echo "Running elm-make with sysconfcpus -n 2"' '$TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 elm-old "$@"' > $(npm config get prefix)/bin/elm
   - chmod +x $(npm config get prefix)/bin/elm
   - npm install
-  - pushd tests && elm make --yes && popd
+  - pushd tests && elm make && popd
```
Comment (turboMaCk) [inline .travis.yml]: I think this should not be part of install. install gets cached this shouldn't

Comment (rofrol) [reply .travis.yml]: What should not be part of install? `elm-make`?

Comment (kraklin) [reply .travis.yml]: yes, formally it is not an install, it is a build, move it to `script` part ;)

File: package.json [hunk 5]
```diff
@@ -1,20 +1,20 @@
...
   "repository": {
     "type": "git",
     "url": "https://github.com/GlobalWebIndex/quantify/"
   },
   "author": "GWI",
   "license": "BSD-3-Clause",
   "scripts": {
-    "test": "yarn elm-verify-examples --elm-test node_modules/.bin/elm-test && yarn elm-test",
-    "test:w": "node_modules/chokidar-cli/index.js \"**/*.elm\" -c \"yarn test\""
+    "test": "npx elm-verify-examples --elm-test=./node_modules/.bin/elm-test && npx elm-test",
```
Comment (turboMaCk) [inline package.json:11]: why `npx`. I think we've had a discussion and agreed we're going to use yarn (I was the only one skeptical).

Comment (rofrol) [reply package.json:11]: npx is very handy tool to make cross-platform differences less painful. It is part of npm@5.2.0, and npm is installed with node.

Regarding yarn, it wasn't working for me.

I can try but why if npm is already on travis? Wouldn't that increase CI time?

Comment (kraklin) [reply package.json:11]: well not sure what did you try but `yarn elm-verify-examples` can't work. Without `yarn` it should work. Think of `yarn` as `npm` - you don't do `npm elm-verify-examples`. But when `npx` works on cross-platform, that's probably better

Comment (turboMaCk) [reply package.json:11]: there are many very handy tools. Problem is that now everyone who is going to do something here needs some other dependency.

Comment (Janiczek) [reply package.json:11]: `yarn elm-verify-examples` does roughly the same as `npm run-script elm-verify-examples`. ie. it adds the node modules .bin to $PATH etc., all that magic. I think it should work.
